<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInstallDialog</class>
 <widget class="QDialog" name="GameInstallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Installiation</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="mp_table_tasks">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Overall progress</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="mp_progressbar_overall">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current progress</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="mp_progressbar_current">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mp_btn_add">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_btn_install">
       <property name="text">
        <string>&amp;Install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="mp_buttonbox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_btn_add</tabstop>
  <tabstop>mp_btn_install</tabstop>
  <tabstop>mp_table_tasks</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mp_buttonbox</sender>
   <signal>accepted()</signal>
   <receiver>GameInstallDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_buttonbox</sender>
   <signal>rejected()</signal>
   <receiver>GameInstallDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_btn_install</sender>
   <signal>clicked()</signal>
   <receiver>GameInstallDialog</receiver>
   <slot>install()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_btn_add</sender>
   <signal>clicked()</signal>
   <receiver>GameInstallDialog</receiver>
   <slot>addTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTask()</slot>
  <slot>install()</slot>
 </slots>
</ui>
